scripts/docker/Dockerfile.sycl

```diff
@@ -26,3 +28,20 @@ RUN CMAKE_KEY=2D2CEF1034921684 && \
     sh ${CMAKE_SCRIPT} --skip-license --prefix=${CMAKE_DIR} && \
     rm cmake*
 ENV PATH=${CMAKE_DIR}/bin:$PATH
+
+ENV SYCL_DIR=/opt/sycl
+RUN SYCL_VERSION=a4f092417ef9d311695b5e2bb600cdb9e2cc4a9a && \
```

Comment:
Why did you go with an explicit commit hash?  The repo seems to have daily tags.

Reply:
I wasn't quite sure if they mirror the `sycl` branch but that seems to be the case looking at it again. The commit is part of `https://github.com/intel/llvm/releases/tag/20201119`. Do you want me to change?

Reply:
Yes I would prefer one of the daily tags.

Reply:
See [`c854bdc` (#3619)](https://github.com/kokkos/kokkos/pull/3619/commits/c854bdcb7fbb60d6422e0846fa9a0818ee40d20e).